config/crd/policy.networking.k8s.io_baselineadminnetworkpolicies.yaml

```diff
@@ -1,10 +1,9 @@
----
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/2522
-    controller-gen.kubebuilder.io/version: v0.11.1
+    api-approved.kubernetes.io: https://github.com/kubernetes-sigs/network-policy-api/issues/90
+    policy.networking.k8s.io/bundle-version: ""
```

Comment:
shouldn't the bundle-version value be "0.1.0" ? 

(nvm) later saw https://github.com/kubernetes-sigs/network-policy-api/pull/93/commits/fac2c0ee8630bc15e59c7b3a6bdb9fc2fffa95bb